test/integration/util/util.go

```diff
@@ -225,6 +225,25 @@ func AddTaintToNode(cs clientset.Interface, nodeName string, taint v1.Taint) err
 	return err
 }
 
+// RemoveTaintOffNode removes taints off a specific node
+func RemoveTaintOffNode(cs clientset.Interface, nodeName string, taint v1.Taint) error {
+	node, err := cs.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	copy := node.DeepCopy()
+	taints := []v1.Taint{}
```

Comment:
`var taints []v1.Taint` is more conventional. Ref: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices